modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/upgrade/DEServiceUpgrade.java

```diff
@@ -63,6 +63,11 @@ public void register(Registry registry) {
 
 			},
 			new UpgradeCTModel("DEDataDefinitionFieldLink", "DEDataListView"));
+
+		registry.register(
+			"2.2.0", "2.2.1",
+			new com.liferay.data.engine.internal.upgrade.v2_2_1.
+				DEDataDefinitionFieldLinkUpgradeProcess());
```

Comment:
@linolaoj, I was checking the 7.3.x code and I realized that the Upgrade Step 2.2.0 isn't available there, in light of this perhaps you'll need to use your previous solution, what do you think? Another alternative would be also backport the Upgrade Step 2.2.0, but it seems to be out of scope of the issue you're fixing.